README.md

```diff
@@ -1,13 +1,13 @@
-# prebid-fledge
-An exploration of the role Prebid can play in a Fledge world.
+# prebid-parakeet
+An exploration of the role Prebid can play in a Parakeet world.
```

Comment:
You should insert a ToC here with all of the links to the headers to make it easier to navigate

Reply:
is there a shortcut to doing that or does it need to be compiled manually?